GNSSLogger/app/src/main/java/com/google/android/apps/location/gps/gnsslogger/FileLogger.java

```diff
@@ -163,6 +165,31 @@ public void startNewLog() {
         currentFileWriter.newLine();
         currentFileWriter.write(COMMENT_START);
         currentFileWriter.newLine();
+        currentFileWriter.write(COMMENT_START);
+        currentFileWriter.write("accelerometer,timestamp,x,y,z");
+        currentFileWriter.newLine();
+        currentFileWriter.write(COMMENT_START);
+        currentFileWriter.newLine();
+        currentFileWriter.write(COMMENT_START);
+        currentFileWriter.write("gyroscope,timestamp,x,y,z");
```

Comment:
These new headers should match the formatting of the others: first letter capitalized. Also, be sure to include units in all the new headers. For example timestampMillis (if it's in milliseconds, nanos if it's in nanoseconds, etc) and the measurement values themselves can be [X,Y,Z]GyroRateRadPerSec, respectively.